manifests/base/config/argocd-cm.yaml

```diff
@@ -5,3 +5,53 @@ metadata:
   labels:
     app.kubernetes.io/name: argocd-cm
     app.kubernetes.io/part-of: argocd
+data:
+  ### Default configuration for ignoreResourceUpdates.
+  # The ignoreResourceUpdates list contains K8s resource's properties that are known to be frequently updated
+  # by controllers and operators. These resources, when watched by argo, will cause many unnecessary updates.
+  # Ignoring status for all resources. An update will still be sent if the status update causes the health to change.
+  resource.customizations.ignoreResourceUpdates.all: |
+    jsonPointers:
+      - /status
+  # Some Application fields are generated and not related to the application updates itself
+  # The Application itself is already watched by the controller lister, but this configuration is applied for apps of apps
+  resource.customizations.ignoreResourceUpdates.argoproj.io_Application: |
+    jqPathExpressions:
+      - '.metadata.annotations."notified.notifications.argoproj.io"'
+      - '.metadata.annotations."argocd.argoproj.io/refresh"'
```

Comment:
This is probably a dumb question, but isn't this annotation put on the app if I click refresh in the UI, which triggers the controller to refresh it?

Will this stop refreshes from happening in the UI?

Reply:
The controller has a lister that watches the Application resources independently of the "cluster resource watches"